### PR TITLE
added new function setBeta for logistic regression model builder on daal4py side

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

File: cpp/daal/include/algorithms/logistic_regression/logistic_regression_model_builder.h [hunk 1]
```diff
@@ -68,6 +68,52 @@ class DAAL_EXPORT ModelBuilder
...
+        const size_t nVectorsBeta = _nClasses == 2 ? 1 : _nClasses;
+        _modelPtr->getBeta()->getBlockOfRows(0, nVectorsBeta, data_management::writeOnly, pBlock);
+        beta->getBlockOfRows(0, nVectorsBeta, data_management::readOnly, pBlockBeta);
+        modelFPType * sp      = pBlock.getBlockPtr();
+        modelFPType * spBeta  = pBlockBeta.getBlockPtr();
+        const size_t betaSize = beta->getNumberOfRows() * beta->getNumberOfColumns();
+        if (betaSize == _nFeatures * nVectorsBeta)
+        {
+            setInterceptFlag(false);
+            for (size_t i = 0, j = 0; j < betaSize; ++j, ++i)
```
Comment (PetrovKP) [inline cpp/daal/include/algorithms/logistic_regression/logistic_regression_model_builder.h:88]:
```c++
for (size_t i = 0; i < nVectorsBeta; ++i)
{
     _modelPtr->getBeta()->getBlockOfRows(i, 1, data_management::writeOnly, resBeta);
     beta->getBlockOfRows(i, 1, data_management::readOnly, pBlockBeta);
     modelFPType * const resBetaData = resBeta.getBlockPtr();
     const modelFPType * const  inBetaData  = inBeta.getBlockPtr();
     resBetaData[0] = modelFPType(0);
     for (size_t j = 0; j < _nFeatures; ++j)
     {
         resBetaData[i + 1] = inBetaData[i];
...
```

**Comment by OnlyDeniko** [comment]
https://ecmd.inn.intel.com/commander/link/jobDetails/jobs/eb0f8a1a-896f-f10d-9fbe-8cdcd4b723a1?firstPage=1

**Comment by OnlyDeniko** [comment]
http://intel-ci.intel.com/eb0f83d6-d779-f134-a89a-8cdcd4b723a1

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**